package.json

```diff
@@ -14,7 +14,7 @@
   },
   "main": "lib/index.js",
   "exports": {
-    ".": "./lib/index.js",
+    ".": { "require": "./lib/index.js", "types": "./typings/index.d.ts" },
     "./types": "./types.js"
```

Comment:
```suggestion
    "./types": { "types": "./typings/types.d.ts", "require": "./lib/types.js" },
```

Reply:
> Note that I didn't change the "./types" exports entry because this seems to be only a workaround for some ESLint error, see https://github.com/telegraf/telegraf/commit/8ddeba23dba69e56f4da6aabdd14417ce7afeb1a.